frontend/common/src/components/Pending/Loading.tsx

```diff
@@ -38,26 +43,28 @@ const Loading = ({ inline = false, live }: LoadingProps): JSX.Element => {
       {...(live && {
         "aria-live": live,
       })}
-      data-h2-display="b(flex)"
-      data-h2-align-items="b(center)"
-      data-h2-justify-content="b(center)"
+      data-h2-display="base(flex)"
+      data-h2-align-items="base(center)"
+      data-h2-justify-content="base(center)"
       style={{
         backgroundColor: inline ? undefined : `rgba(255,255,255,0.95)`,
         bottom: 0,
-        left: 0,
-        right: 0,
+        left: "1rem",
+        right: "1rem",
```

Comment:
This is causing a noticeable gap on either side of the screen while loading.

<img width="871" alt="Screen Shot 2022-08-22 at 2 17 50 PM" src="https://user-images.githubusercontent.com/4127998/185992800-d46edf19-7ccc-4e85-83d3-eab75f17df3f.png">

Reply:
Oh damn, good catch. I was totally looking at this in Storybook and missed it there. @esizer Do you think it's easy to apply those values exclusively to the inline loader?

Reply:
Should be something along the lines of:

```tsx
style={{
  ...
  left: inline ? `1rem` : 0,
  right: inline ? `1rem` : 0,
  ...
 }}
```

I can push this up if you'd like.

Reply:
Yeah, that would be great!

Reply:
Resolved with [fix spacing on full screen loading spinner](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3051/commits/9a4008e965d9a790d8aa5f1e26b48ad6b7593f61)